general-superstaq/general_superstaq/check/check_utils.py

```diff
@@ -13,10 +13,13 @@
 from typing import Any, Callable, Iterable, List, Union
 
 # identify the root directory of the "main" script that called this module
-main_file_dir = os.path.dirname(os.path.abspath(sys.argv[0]))
-root_dir = subprocess.check_output(
-    ["git", "rev-parse", "--show-toplevel"], cwd=main_file_dir, text=True
-).strip()
+try:
+    main_file_dir = os.path.dirname(os.path.abspath(sys.argv[0]))
+    root_dir = subprocess.check_output(
+        ["git", "rev-parse", "--show-toplevel"], cwd=main_file_dir, text=True
+    ).strip()
+except subprocess.CalledProcessError:
+    root_dir = subprocess.check_output(["git", "rev-parse", "--show-toplevel"], text=True).strip()
```

Comment:
(this change just makes it so we can import `general_superstaq.check` in a python terminal, which is really nice for debugging)